regional_rail_equity/analysis/summarize_stations_tod.py

```diff
@@ -0,0 +1,32 @@
+from regional_rail_equity import db
+from regional_rail_equity.database.config.station_summary_config import (
+    station_summary_files,
+)
+
+query_template = """
+    drop table if exists NEW_TABLENAME;
+
+    create table NEW_TABLENAME as
+    
+    with joined_model_with_tod as(
+        select *, st_transform(a.geom, 4326) 
```

Comment:
you should alias `st_transform(a.geom, 4326)` with an `AS geom`. This may require listing out column to `SELECT` instead of `*` so you don't also have the old `geom` column